public/tpl/header.html

```diff
@@ -44,7 +44,7 @@
                 <li class="active">
                     <a href="/">Home</a>
                 </li>
-                <li ng-show="settings.showRichList">
```

Comment:
why remove this line without proper explanation?

one can disable `settings.showRichList = false` while `$ node tools/richlist.js` sync or any richlist maintaining step.

Reply:
Is there any reason to disable them while it is being synced by default? ( using `tools/sync.js` )

Reply:
Addressed your concern @hackmod , thanks for reviewing.